cmd/prysmctl/withdrawal/cmd.go

```diff
@@ -0,0 +1,56 @@
+package withdrawal
+
+import (
+	"os"
+
+	"github.com/prysmaticlabs/prysm/v3/cmd"
+	"github.com/prysmaticlabs/prysm/v3/runtime/tos"
+	log "github.com/sirupsen/logrus"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	BeaconHostFlag = &cli.StringFlag{
+		Name:  "beacon-node-host",
+		Usage: "host:port for beacon node to query",
+		Value: "127.0.0.1:3500",
+	}
+	PathFlag = &cli.StringFlag{
+		Name:    "path",
+		Aliases: []string{"p"},
+		Usage:   "path to the signed withdrawal messages JSON",
+	}
+	SkipPromptsFlag = &cli.BoolFlag{
+		Name:  "skip-prompts",
+		Usage: "DANGER:User accepts responsibility of all input data without prompt verification! skips all prompts for verifying each provided signed withdrawal message to for setting the withdrawal address",
+	}
+)
+
+var Commands = []*cli.Command{
+	{
+		Name:    "set-withdrawal-addresses",
```

Comment:
I think this is in a subcommand called `withdrawal`, so maybe you could call this `set-address` for conciseness?

Reply:
it's not in a sub command right now so that's why it started as that, withdrawal doesn't have any other commands but perhaps it should and have validator exit as another sub command?

Reply:
Actually I think this isn't the case, so I guess the top-level prysmctl command will have a subcommand called `set-withdrawal-address`. This feels a bit clunky, so I wonder if there is a logical grouping for this to be under another subcommand to keep things organized. maybe something like `prysmctl validator withdraw`, and then we could compliment that with `prysmctl validator exit`?

Reply:
I think that works I'll refactor that then